DEVELOPING_OPAL/tools/src/main/scala/org/opalj/support/info/Purity.scala

```diff
@@ -616,16 +639,12 @@ object Purity {
 
         val purityConf = new PurityConf(args)
 
-        if (args.contains("--help") || args.contains("-h")) {
-            return
-        }
-
         val begin = Calendar.getInstance()
         Console.println(begin.getTime)
 
         time {
             if (purityConf.multiProjects.get) {
-                for (subp <- purityConf.classPathFiles.get.head.listFiles().filter(_.isDirectory)) {
+                for (subp <- purityConf.classPathFiles.get.flatMap(_.listFiles).filter(_.isDirectory)) {
```

Comment:
Instead of a flatMap, make this two iterations (see https://docs.scala-lang.org/tour/for-comprehensions.html for an example)